fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1048,6 +1050,10 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = ENABLE_CONNECTOR_SINK_GLOBAL_SHUFFLE, flag = VariableMgr.INVISIBLE)
     private boolean enableConnectorSinkGlobalShuffle = true;
 
+
+    @VariableMgr.VarAttr(name = CONNECTOR_SINK_ENABLE_SPILL, flag = VariableMgr.INVISIBLE)
```

Comment:
enable_connector_sink_spill?

Reply:
connector_sink_enable_spill sounds more natural to me.

Reply:
renamed